gui/src/installer/step/mod.rs

```diff
@@ -229,6 +638,71 @@ impl From<DefineBitcoind> for Box<dyn Step> {
     }
 }
 
+impl From<StartInternalBitcoindStep> for Box<dyn Step> {
+    fn from(s: StartInternalBitcoindStep) -> Box<dyn Step> {
+        Box::new(s)
+    }
+}
+
+impl StartInternalBitcoindStep {
+    pub fn new(liana_datadir: &PathBuf) -> Self {
+        Self {
+            conf_path: internal_bitcoind_config_path(liana_datadir),
```

Comment:
Somehow my Liana datadir gets catenated to my Bitcoin datadir
![image](https://github.com/wizardsardine/liana/assets/22457751/c5c644d9-154e-4ac6-ac1c-27b7e20a200d)

Reply:
Hmm, I haven't encountered this error, but will try to see if I'm mixing directories somewhere.